locale.php

```diff
@@ -57,22 +57,46 @@ public function getFileName($id) {
     /**
      *
      */
-    public function getByPrefix($prefix, $strip = true) {
+    public function getLangJSPrefix() {
+        $jslang = array('js_status', 'js_loading', 'js_loaded',
+                        'js_loading_failed', 'js_invalid_data',
+                        'js_saving', 'js_saved', 'js_saving_failed',
+                        'js_invalid_ns_name', 'js_ns_name_exists',
+                        'js_delete_ns', 'js_invalid_note_name',
+                        'js_note_name_exists', 'js_delete_note',
+                        'js_unsaved');
```

Comment:
Having this list in multiple places (lang file and here) is error-prone. Yes, it hardly ever is updated, but still.